entrypoint.sh

```diff
@@ -122,16 +122,17 @@ function plan() {
     if [[ $ACTION == "Destroy" ]]; then
         PLAN_EXTRA_OPTIONS="-destroy"
     fi
-    $TERRAFORM_CMD plan ${PLAN_EXTRA_OPTIONS} -out=${PLAN_FILE} ${TERRAFORM_VARS} ${LOCK}
-    $TERRAFORM_CMD show -json ${PLAN_FILE} > ${PLAN_FILE_JSON}
+    $TERRAFORM_CMD plan ${PLAN_EXTRA_OPTIONS} -out="${PLAN_FILE}" ${TERRAFORM_VARS} ${LOCK}
+    $TERRAFORM_CMD show -json "${PLAN_FILE}" > "${PLAN_FILE_JSON}"
     run_hook "post_plan"
 }
 
 function apply() {
     run_hook "pre_apply"
     if [[ $ACTION == "Apply" ]] && [[ $DRY_RUN == "False" ]]; then
-        $TERRAFORM_CMD apply -auto-approve ${PLAN_FILE}
-        $TERRAFORM_CMD output -json > $OUTPUTS_FILE
+        $TERRAFORM_CMD apply -auto-approve "${PLAN_FILE}"
```

Comment:
`-auto-approve` not needed for [Saved Plan Mode](https://developer.hashicorp.com/terraform/cli/commands/apply#saved-plan-mode), should we also use saved plan when `destroy`?

Reply:
Changed in f3652ef